console_backend/src/cli_options.rs

```diff
@@ -199,6 +199,10 @@ pub struct CliOptions {
     /// Set the width of the main window.
     #[clap(long)]
     pub width: Option<u32>,
+
+    /// Disable ntrip client
+    #[clap(long)]
+    pub disable_ntrip: bool,
```

Comment:
This needs to be inverted to `--enable-ntrip` -- we want a way to force ntrip to be enabled so that we can test the client with other setups (such as chaining socat instances together).

Reply:
By default the ntrip client will be disabled unless we detect that we're connect to a PT device.